addons/digest/tests/test_digest_kpi.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.digest.tests.common import TestDigestCommon
+
+
+class TestDigest(TestDigestCommon):
+
+    def test_kpi_mail_message_total_value(self):
```

Comment:
There is already a test for that in ``test_digest.py`` (could move ``_setup_messages`` in common by the way)

Reply:
We could also use the common class in ``test_digest.py`` by the way